### ISSUE TITLE
Crash On Save

### ISSUE THREAD

**Issue by SenselessMite** [KuromeSan/chovy-sign#6]
Hi, I have a problem with Monster Hunter Freedom United. The game works properly, but when I save the game crashes and it wasn't boot anymore. I need to delete the save file to play the game again. 
The error code is C1-2858-3

**Comment by KuromeSan** [comment]
Does it work in adrenaline ? Think maybe its pspemu issue 

Also u can delete PSP savedata in content mannager. Just so ya know ;)

**Comment by SenselessMite** [comment]
I can't install adrenaline because my firmware is 3.71.

**Comment by KuromeSan** [comment]
Oh 3.71.. right of course. 

Hmm can you send me the ISO so I can try?

**Comment by SenselessMite** [comment]
Yes,  I'm uploading it to google drive. When it has finished I will share the download link for you

**Comment by SenselessMite** [comment]
Sorry for the delay. I have loaded the iso to drive, this is the link
https://drive.google.com/open?id=1uymNtDw9NueAd5BnoyhcBO4Uq6JaRKHw

**Comment by nmagod** [comment]
The redump.org ISO works for me, but at what point exactly does it crash for you? Saving when you make a new character, or saving after you've completed a quest?

**Comment by SenselessMite** [comment]
Every time I save after I've already created a character or when I close the game and re-started it

**Comment by nmagod** [comment]
Check your ISO against redump's information http://redump.org/disc/8470/
if it doesn't match, *get* the redump ISO and try it, after deleting your save (and the install data, if you happened to enable that at any point)

yes, I know it's almost 800MB
